src/main/kotlin/lotto/model/GenerateLotto.kt

```diff
@@ -0,0 +1,14 @@
+package lotto.model
+
+/**
+ * 로또 생성하는 것을 관리하는 클래스
+ * */
+class GenerateLotto(private val price: Price) {
+    fun generateLottoList(): List<Lotto> =
+        (0 until price.lottoCount)
+            .map {
+                Lotto(GenerateRandomInt.generateNumbers(LottoNumber.getRandomRangeList())
```

Comment:
```suggestion
                Lotto(item = GenerateRandomInt.generateNumbers(range = LottoNumber.getRandomRangeList())
```
로또 객체의 생성자 및 generateNumbers 함수의 파리미터에 어떤 것이 들어가는지 알기가 어렵네요
이런 경우에는 argument name을 활용하면 가독성이 좋아집니다